docs/BUILD.md

```diff
@@ -134,6 +134,28 @@ For help building a desired libcrypto on your platform, please consult the proje
 
 The `CMAKE_INSTALL_PREFIX` option can be provided when building AWS-LC to specify where AWS-LC will be installed. The install path for AWS-LC should be provided when building s2n-tls, via the `CMAKE_PREFIX_PATH` option. This will ensure that s2n-tls is able to find the AWS-LC library artifact to link with.
 
+### Openssl FIPS
+
+If you require FIPS, you should consider using AWS-LC. If AWS-LC would conflict with existing Openssl in your environment, you can use the `S2N_INTERN_LIBCRYPTO` CMake option to "intern" AWS-LC and keep it isolated to s2n-tls.
```

Comment:
nit: I think "existing" as written doesn't sound right. Maybe it can be removed?
```suggestion
If you require FIPS, you should consider using AWS-LC. If AWS-LC would conflict with Openssl in your environment, you can use the `S2N_INTERN_LIBCRYPTO` CMake option to "intern" AWS-LC and keep it isolated to s2n-tls.
```